src/main/scala/com/webtrends/harness/component/akkahttp/AkkaHttpBase.scala

```diff
@@ -63,11 +64,25 @@ trait AkkaHttpBase extends PathDirectives with MethodDirectives with AccessLog w
 
   // The AkkaHttpCORS trait is provided to enable CORS if desired
   protected val corsEnabled: Boolean = false
+  def allOrigins: Seq[HttpOrigin] =Seq()
```

Comment:
Is the intention that this is overridden by a class extending AkkaHttpBase for the allowed origins?

If so, the name "all" is misleading. Can we name it "allowedOrigins" instead?